ldms/src/ldmsd/ldmsd_request.c

```diff
@@ -5498,15 +5498,16 @@ struct op_summary {
 	uint64_t op_mean_us;
 };
 
+#define __ROUND(X) ((((X)-1)|0xFFF)+1) /* round-up to multiples of 4K */
```

Comment:
@narategithub,

Coudl we just add this to ldms.h or something so we can use it everywhere.
```C
#define LDMS_ROUNDUP(_sz_, _align_) (((_sz_) + (_align_) - 1) & ~((_align_)-1))
```

Reply:
OK.